tests/e2e/Services/Account/AccountBase.php

```diff
@@ -308,6 +308,8 @@ public function testGetAccountSessions($data): array
 
         $this->assertEquals(true, $response['body']['sessions'][0]['current']);
 
+        $this->assertEquals(true, $response['body']['sessions'][0]['expire']);
```

Comment:
Te expected value should be a valid ISO Formatted date string, not true

Reply:
Have fixed this